src/app/portlets/content-types/fields/content-type-fields-drop-zone/content-type-fields-drop-zone.component.ts

```diff
@@ -11,18 +11,56 @@ import {
 } from '@angular/core';
 import { FieldDragDropService, DropFieldData } from '../service';
 import { FieldType } from '../models';
-import { DotCMSContentTypeField, DotCMSContentTypeLayoutRow } from 'dotcms-models';
+import {
+    DotCMSContentTypeField,
+    DotCMSContentTypeLayoutRow,
+    DotCMSContentTypeLayoutColumn
+} from 'dotcms-models';
 import { ContentTypeFieldsPropertiesFormComponent } from '../content-type-fields-properties-form';
 import { DotMessageService } from '@services/dot-messages-service';
-import { FieldUtil } from '../util/field-util';
+import { FieldUtil, COLUMN_FIELD, COLUMN_BREAK_FIELD } from '../util/field-util';
 import { FieldPropertyService } from '../service/field-properties.service';
 import { DotDialogActions } from '@components/dot-dialog/dot-dialog.component';
 import { DotEventsService } from '@services/dot-events/dot-events.service';
-import { takeUntil, take  } from 'rxjs/operators';
+import { takeUntil, take } from 'rxjs/operators';
 import { Subject } from 'rxjs';
 import { DotLoadingIndicatorService } from '@components/_common/iframe/dot-loading-indicator/dot-loading-indicator.service';
 import * as _ from 'lodash';
 
+const splitRows = (fieldRows: DotCMSContentTypeLayoutRow[]): DotCMSContentTypeLayoutRow[] => {
+    return fieldRows.map((row) => {
+        return {
+            ...row,
+            columns: row.columns.reduce(
+                (acc: DotCMSContentTypeLayoutColumn[], current: DotCMSContentTypeLayoutColumn) => {
+                    const indexOfBreak = current.fields.findIndex((item) => {
+                        return item.clazz === COLUMN_BREAK_FIELD.clazz;
+                    });
+                    if (indexOfBreak > -1) {
+                        const first = current.fields.slice(0, indexOfBreak);
+                        const second = current.fields.slice(indexOfBreak + 1);
+
+                        const firstCol = {
+                            ...current,
+                            fields: first
+                        };
+
+                        const secondCol = {
+                            columnDivider: COLUMN_FIELD,
```

Comment:
here we should use this method

https://github.com/dotCMS/core-web/blob/8cc483ffd528d0067e3e811fb801b984541c8e14/src/app/portlets/content-types/fields/util/field-util.ts#L127